eng/pipelines/stages/preparerelease.yml

```diff
@@ -1,84 +1,84 @@
-          workingDirectory: '$(Build.Repository.LocalPath)'
-      - script: >-
-          dotnet.cmd run --project $(Build.Repository.LocalPath)\eng\release\DiagnosticsReleaseTool\DiagnosticsReleaseTool.csproj -c Release
-          --
-          prepare-release 
-          --input-drop-path "$(System.ArtifactsDirectory)\ReleaseTarget" 
-          --tool-manifest "$(Build.Repository.LocalPath)\eng\release\tool-list.json" 
-          --staging-directory "$(System.ArtifactsDirectory)\ReleaseStaging" 
-          --release-name "$(Build.BuildNumber)" 
-          --account-name "$(dotnet-diagnostics-storage-accountname)" 
-          --account-key "$(dotnetstage-storage-key)" 
-          --sas-valid-days "$(dotnet-diagnostics-storage-retentiondays)" 
-          -v True
-        workingDirectory: '$(Build.Repository.LocalPath)\'
-        displayName: 'Manifest generation and asset publishing'
-      - task: PublishPipelineArtifact@1
-        inputs:
-          targetPath: '$(System.ArtifactsDirectory)\ReleaseStaging'
-          publishLocation: 'pipeline'
-          artifact: 'MonitorRelease'
-      - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), or(startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), startsWith(variables['Build.SourceBranch'], 'refs/heads/internal/release/'))) }}:
-        - task: tagBuildOrRelease@0
-          displayName: 'Tag Build with Monitor Release'
-          inputs:
-            type: 'Build'
-            tags: 'MonitorRelease'
+stages:
+- stage: PrepareReleaseStage
+  displayName: Release Preparation
+  dependsOn:
+  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), or(startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), startsWith(variables['Build.SourceBranch'], 'refs/heads/internal/release/'), startsWith(variables['Build.SourceBranch'], 'refs/heads/test/release/'))) }}:
+    - publish_using_darc
+  jobs:
+  - job: PrepareReleaseJob
+    displayName: Prepare release with Darc
+    pool: 
+      ${{ if eq(variables['System.TeamProject'], 'public') }}:
+        name: NetCore-Public
+        demands: ImageOverride -equals 1es-windows-2019-open
+      ${{ if ne(variables['System.TeamProject'], 'public') }}:
+        name: NetCore1ESPool-Internal
+        demands: ImageOverride -equals 1es-windows-2019
+    variables:
+    - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), or(startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), startsWith(variables['Build.SourceBranch'], 'refs/heads/internal/release/'), startsWith(variables['Build.SourceBranch'], 'refs/heads/test/release/'))) }}:
+      - group: DotNet-Diagnostics-Storage
+      - group: DotNet-DotNetStage-Storage
+      - group: Release-Pipeline
+    steps:
+    - task: UseDotNet@2
+      displayName: 'Use .NET Core runtime 3.1.x'
```

Comment:
Do we still want this?

Reply:
I didn't want to change functionality; I can look into this separately.